backend/app/tests/crud/test_crud_share.py

```
@@ -0,0 +1,293 @@
+        content_item_id=content_item.id,  # Not strictly used by CRUD, but good for schema validation
+        expires_at=expires_at_dt,
+        max_access_count=10,
+        password="testpassword",
+    )
+
+    created_share = crud_content.create_content_share(
+        db=db,
+        content_share_in=share_in,
+        content_item_id=content_item.id,
+        user_id=user.id,  # user_id for audit/context, not stored on ContentShare model itself
+    )
```

Pylint (3.3.7)

[error] Unexpected keyword argument 'user_id' in function call

(E1123)